src/Plutus/Types/CurrencySymbol.purs

```diff
@@ -10,15 +10,25 @@ module Plutus.Types.CurrencySymbol
 
 import Prelude
 
+import Contract.Prelude (Either(..))
+import Data.Argonaut
+  ( class DecodeJson
+  , class EncodeJson
+  , JsonDecodeError(..)
```

Comment:
Usually we use explicit constructor imports

Reply:
IDE does this for some reason like this, terrible I agree, fixing